classes/responsetype/responsetype.php

```diff
@@ -198,7 +198,7 @@ public function get_results_tags($weights, $participants, $respondents, $showtot
             if ($showtotals) {
                 $pagetags->total = new \stdClass();
                 if ($respondents > 0) {
-                    $percent = round((float)$respondents / (float)$participants * 100.0);
+                    $percent = 100;
```

Comment:
Why this change? This changes a calculation to a hard coded value.

Reply:
Hi @mchurchward 
Sorry for this change. During the development of this issue (previously handled by our former developer), it seems that updating the "All participants" text to a dropdown list with the content "Full submissions" - Default value, "All responses", "Responses not submitted" led us to assume that the "Total responses to question" row should always be the sum of the response number/percent above. For example:

Response: Yes - 1 -> 50%
Response: No - 1 -> 50%
Total responses to question row: (2) -> 100%

After thoroughly investigating the logic of this row, we now understand that it should not be hard-coded to 100 as it was before. We will update this code to align with the current logic.